components/omega/doc/devGuide/CondaEnv.md

```diff
@@ -7,45 +7,43 @@ code, you will need your own
 [conda](https://conda.io/projects/conda/en/latest/index.html) environment with
 the necessary packages.
 
-(omega-dev-install-mambaforge)=
+(omega-dev-install-miniforge3)=
 
-## Installing Mambaforge
+## Installing Miniforge3
 
 If you have not already done so, you will install
-[Mambaforge](https://github.com/conda-forge/miniforge#mambaforge), preferably
+[Miniforge3](https://github.com/conda-forge/miniforge#miniforge3), preferably
 somewhere in your home directory but possibly somewhere in your scratch space.
```

Comment:
It may be helpful to add the CL install commands here to potentially save a click into the Miniforge3 link:

```
curl -L -O "https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-$(uname)-$(uname -m).sh"
bash Miniforge3-$(uname)-$(uname -m).sh
```

Reply:
````suggestion
somewhere in your home directory but possibly somewhere in your scratch space:
```bash
curl -L -O "https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-$(uname)-$(uname -m).sh"
bash Miniforge3-$(uname)-$(uname -m).sh
```
````